### ISSUE TITLE
SimplePie 1.8.1 version not updated

### ISSUE THREAD

**Issue by prbt2016** [simplepie/simplepie#904]
Hello @jrfnl  ,

I was in the process of manual installation of SimplePie on Centos 7 with PHP 7.2, Apache 2.2, MYSQL 5.5 using 

`git clone https://github.com/simplepie/simplepie.git`

And followed these steps : 

http://simplepie.org/wiki/setup/setup

However, when i checked the file src/SimplePie.php, it seems that the version is not updated i.e 

`public const VERSION = '1.8.0';` instead of `public const VERSION = '1.8.1';

Could you please check and update this.

**Comment by jtojnar** [comment]
1.8.1 is in `one-dot-eight` branch: https://github.com/simplepie/simplepie/blob/1.8.1/src/SimplePie.php#L73
`master` contains the work on upcoming 1.9 release.

See also https://github.com/simplepie/simplepie#project-status

**Comment by prbt2016** [comment]
Hello @jtojnar ,

Thanks for the quick response . 

So I would have to do git clone -b one-dot-eight https://github.com/simplepie/simplepie.git

For getting the 1.8 branch right ?.

Regards.

**Comment by jtojnar** [comment]
Right. Or you can use `git switch one-dot-eight` in the existing repo. Or switch to `1.8.1` directly if you want the tagged version.

**Comment by prbt2016** [comment]
Okay cool . Thanks for the support.
